### PR TITLE
added logo and uni name in pdf

### PR DESCRIPTION
**Pull request by ziqiang2344**
About to add table to store student ID and course list

### CODE REVIEW

File: src/sample/downloadController.java [hunk 1]
```diff
@@ -37,38 +46,56 @@ public static void inputID(String text){
...
 
       PdfDocument pdfDocument = new PdfDocument(pdfWriter);
       pdfDocument.addNewPage();
 
       Document document = new Document(pdfDocument);
-      document.add(list1);
-
+      document.add(image1);
+      document.add(new Paragraph("Universiti Sains Malaysia").setTextAlignment(TextAlignment.CENTER).setBold().setFontSize(30f));
+      document.add(new Paragraph("Student ID:   "+stuID));
```
Comment (Michelle-Lohwt) [inline src/sample/downloadController.java:107]: Can add name or not?

Comment (Michelle-Lohwt) [reply src/sample/downloadController.java:107]: If the user didn't put the name, just display "User didn't update the name" smthg like that

**Review by Michelle-Lohwt — CHANGES_REQUESTED** [review]
check the comments

**Comment by ziqiang2344** [comment]
> check the comments

when student name is blank


Snag_2281abe
when student name is inserted in the dashboard

File: src/sample/downloadController.java [hunk 2]
```diff
@@ -32,43 +39,75 @@
...
-    if (file != null) {
-      List list1 = new List();
-      // input the text file for the course list
+     if (file != null) {
+
+      System.out.println("Path: " + file.getAbsolutePath());
+      textfield.setText(file.getAbsolutePath());
+
+      //Image
+      ImageData data= ImageDataFactory.create("C:/Users/user/student-course-registration-system/src/sample/images/usm-ringlogo.png");
```
Comment (Michelle-Lohwt) [inline src/sample/downloadController.java]: I think the file name might be incorrect

**Review by Michelle-Lohwt — CHANGES_REQUESTED** [review]
Check the comments

File: src/sample/downloadController.java (hunk 2 above)
Comment (ziqiang2344) [reply src/sample/downloadController.java]: u mean the file path?

**Review by Michelle-Lohwt — APPROVED** [review]

**Pull request merged**